pkg/github/github.go

```diff
@@ -236,8 +268,17 @@ func componentFromRepo(
 
 	log.OutLogger.Printf("  Relevant versions: [%s]", strings.Join(relevantVersions, ", "))
 
+	// Check if there is a "releases/{repo.Version}" branch
+	// If it exists, use that; if not, use master.
+	hasReleaseBranch, err := checkForBranch(httpClient, "github_api", repo.Name, fmt.Sprintf("release/%s", suiteVersion))
```

Comment:
`err` is not checked at all here

Reply:
Order of code here is a bit odd - we probably want to set branch to master first, then do the check for release branch

Reply:
fixed.